cmd/veneur-emit/main.go

```diff
@@ -65,6 +66,8 @@ var (
 	// Tracing flags
 	traceID   = flag.Int64("trace_id", 0, "ID for the trace (top-level) span. Setting a trace ID activated tracing.")
 	parentID  = flag.Int64("parent_span_id", 0, "ID of the parent span.")
+	spanStart = flag.String("span_starttime", "", "Date/time to set for the start of the span. See https://github.com/araddon/dateparse#extended-example for formatting.")
```

Comment:
Why are these stored in variables but not used?

Reply:
Hah, that's a great question. i suspect it's probably a bit of sloppiness over time - we started with the simple pattern of declaring these package-level vars for the flags, but then introduced the `flags()` func and the `passedFlags` abstraction, which obviated the need for referencing these package vars directly.

They could probably all be removed now. For now, though, IMO consistency requires that we add them, and then a subsequent PR could drop them.